src/main/scala/edu/ie3/osmogrid/ActorStopSupportStateless.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * © 2022. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.osmogrid
+
+import akka.actor.typed.Behavior
+import akka.actor.typed.scaladsl.Behaviors
+import org.slf4j.Logger
+
+/** Support trait for executing clean up tasks with stopping an actor
+  * @tparam T
+  *   Behavior type
+  */
+trait ActorStopSupportStateless[T] {
```

Comment:
Why did you decide to not extend the existing trait and override the methods?

Reply:
I now adapted the interface definitions to match types as far as possible (both using functions now)